### PR TITLE
Ajax login

### PR DESCRIPTION
**Pull request by Phhere**
When javascript is enabled the login form is submitted from ajax so the checklogin page is not visible to the user.
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/derchrisuk%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/pull/204%23issuecomment-73769994%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-02-10T19%3A48%3A05Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/706758%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/derchrisuk%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20155de6027f9cb2e01c10f3407e3bf0e8500f0827%20js/bbcode.js%208%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL/pull/204%23discussion_r24444788%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22use%20%60form%5Bname%3Dlogin%5D%60%20so%20we%20don%27t%20have%20to%20add%20an%20id%20to%20the%20form%3F%22%2C%20%22created_at%22%3A%20%222015-02-10T19%3A55%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-02-17T19%3A27%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20js/bbcode.js%3AL699-755%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/webSPELL/webSPELL/pull/204%23issuecomment-73769994%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/204%23discussion_r24444788%22%2C%20%22https%3A//github.com/webSPELL/webSPELL/pull/204%23discussion_r24845244%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/derchrisuk'></a>

- [x] <a href='#crh-comment-Pull 155de6027f9cb2e01c10f3407e3bf0e8500f0827 js/bbcode.js 8'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL/pull/204#discussion_r24444788'>File: js/bbcode.js:L699-755</a></b>
- <a href='https://github.com/Pascalmh'></a> use `form[name=login]` so we don't have to add an id to the form?

<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/204?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/204?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL/pull/204?approve=0'></a>&nbsp;<a href='https://github.com/webSPELL/webSPELL/pull/204'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by webSPELL-org** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/webSPELL/webSPELL/pull/204'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by derchrisuk** [comment]
:shipit:

File: js/bbcode.js [hunk 1]
```diff
@@ -699,3 +699,56 @@ if (typeof calledfrom === "undefined") {
 } else if (calledfrom === "admin") {
     fetch("../getlang.php?modul=bbcode&mode=array", "none", "execute", "event");
 }
+
+$(document).ready(function()
+    {
+    "use strict";
+    $("form#login").submit(function(e)
```
Comment (Pascalmh) [inline js/bbcode.js]: use `form[name=login]` so we don't have to add an id to the form?

Comment (Pascalmh) [inline js/bbcode.js]: :+1:

**Pull request merged**